src/transport_ofi.h

```diff
@@ -1153,29 +1170,39 @@ static inline
 uint64_t shmem_transport_received_cntr_get(void)
```

Comment:
The changes to the received_cntr_get/wait functions seem weird, since these functions already generate progress.  IIRC, these changes were part of a hack to use these routines as stand-ins for a proper "probe" routine during earlier exploratory work.  We do need to respect the received_cntr lock here, but we should be using "probe" to generate progress instead of using these routines.

Reply:
Oh yeah - it looks like I didn't see that using these routines earlier was just a hack for probe.  I'll just back out of these changes.